src/Exception/InvalidReleaseVersion.php

```diff
@@ -31,16 +31,16 @@ class InvalidReleaseVersion extends \LogicException implements ExceptionInterfac
 	public static function fromVersion(
 		string $version,
 		string $package_name,
-		$code = null,
+		int $code = null,
```

Comment:
It doesn't look like the `$code` can default to `null`. If I run `$package->get_release( '0.0.0' )`, PHP throws a fatal error:

`Fatal error: Uncaught Error: Wrong parameters for SatisPress\Exception\InvalidReleaseVersion([string $message [, long $code [, Throwable $previous = NULL]]]) in /srv/www/local.test/public/content/plugins/satispress/src/Exception/InvalidReleaseVersion.php:39`

Reply:
Thanks - I'll look into this and update as necessary.